### PR TITLE
New proposed logo

### PR DESCRIPTION
**Pull request by ska-ibees**
To match logo and content with documentation README.md

### CODE REVIEW

File: _pages/terradoocloud.md [hunk 1]
```diff
@@ -6,65 +6,68 @@ permalink: /
 {{ content }}
 
 
-What Is ?
+
+
+ 
+![Terradoo Cloud](source/_themes/images/Terradoo-cloud-new-logo.png)
```
Comment (JuanDCG) [inline _pages/terradoocloud.md:12]: When we decide a new logo change it name without "new", to overwrite old image and image path name.

File: _pages/terradoocloud.md [hunk 2]
```diff
@@ -6,65 +6,68 @@ permalink: /
...
+
+
+What is Terradoo Cloud ?
 =========
 
-  The main objective of Terradoo Cloud is move Odoo to Multi Cloud Native.
+  The main objective of Terradoo Cloud is to transform Odoo into a MultiCloud-Native Business suite.
 
-  With an approach based on microservices it intents to build a solution for Odoo Community on a Multi Cloud Native Architecture.
+  With an approach based on microservices it intents to build a solution for Odoo Community on a MultiCloud-Native Architecture.
```
Comment (JuanDCG) [inline _pages/terradoocloud.md:20]: Multi Cloud Native without any dash

File: _pages/terradoocloud.md [hunk 3]
```diff
@@ -6,65 +6,68 @@ permalink: /
...
+  OpenERP is a well-designed software but it inherits a monolithic approach.
 
-  Luckely, it's designed with MVC web server patron and it has a powerfull ORM with a great database together.
+  Luckily, it's designed with MVC web server pattern and it has a powerful ORM with a tight database integration.
 
-  So, ideally, it could be possible split into microservices Odoo itself. But, it isn't the main goal of this project.
+  So, ideally, it could be possible to split Odoo into microservices. But, it isn't the main goal of this project.
 
-  However, OpenERP/Odoo need other services to delivery cloud applications nowdays that could be integrate easily together.
+  However, an OpenERP/Odoo based project may needs other supporting services/applications. Terradoo cloud aims to provide a seamless solution to bundle such a project as a MultiCloud-Native application.
```
Comment (JuanDCG) [inline _pages/terradoocloud.md:28]: Multi Cloud Native without any dash

File: _pages/terradoocloud.md [hunk 4]
```diff
@@ -82,14 +85,22 @@ Project Management
...
   4. If you are amazed with the project, I'll invite you as outside collaborator to the [Organization](https://github.com/TerradooCloud) or we can negotiate access level to the [Organization](https://github.com/TerradooCloud).
 
-  5. If you prefer, you can reach me by [email](mailto:contact@terradoo.cloud). I'll be excited.
\ No newline at end of file
+  5. If you prefer, you can reach me by [email](mailto:contact@terradoo.cloud). I'll be excited.
+
+
+
+
+License
```
Comment (JuanDCG) [inline _pages/terradoocloud.md:103]: License notice on website is in other place.

File: _pages/terradoocloud.md (hunk 1 above)
Comment (ska-ibees) [reply _pages/terradoocloud.md:12]: ok, i thought to give it a new name to avoid cache problem. Anyways, will keep it same in future.

Comment (JuanDCG) [reply _pages/terradoocloud.md:12]: Ok, The Original Logo was build in no more than 5 minutes, so I've it on my local station, and there isn't problem to overwrite on github.

Comment (ska-ibees) [reply _pages/terradoocloud.md:12]: Agreed. Will keep single file with same name.

File: _pages/terradoocloud.md (hunk 2 above)
Comment (ska-ibees) [reply _pages/terradoocloud.md:20]: Do you want me to make this change and create a new PR?

Comment (JuanDCG) [reply _pages/terradoocloud.md:20]: Yes, right

**Comment by ska-ibees** [comment]
Check #3

**Pull request merged**